fury/actor.py

```diff
@@ -2720,12 +2727,61 @@ def billboard(
     bb_actor.GetProperty().BackfaceCullingOff()
     attribute_to_actor(bb_actor, big_centers, 'center')
 
+    bb_norm = import_fury_shader(os.path.join('utils',
+                                              'billboard_normalization.glsl'))
+
+    if bb_type.lower() == 'cylindrical_x':
+        bb_type_sd = import_fury_shader(os.path.join('billboard',
+                                                     'cylindrical_x.glsl'))
+        v_pos_mc = \
+            """
+            vec3 vertexPositionMC = cylindricalXVertexPos(center, MCVCMatrix,
+                                        normalizedVertexMCVSOutput, shape);
+            """
+    elif bb_type.lower() == 'cylindrical_y':
+        bb_type_sd = import_fury_shader(os.path.join('billboard',
+                                                     'cylindrical_y.glsl'))
+        v_pos_mc = \
+            """
+            vec3 vertexPositionMC = cylindricalYVertexPos(center,MCVCMatrix,
+                                        normalizedVertexMCVSOutput, shape);
+            """
+    elif bb_type.lower() == 'spherical':
+        bb_type_sd = import_fury_shader(os.path.join('billboard',
+                                                     'spherical.glsl'))
+        v_pos_mc = \
+            """
+            vec3 vertexPositionMC = sphericalVertexPos(center, MCVCMatrix,
+                                        normalizedVertexMCVSOutput, shape);
+            """
+    else:
+        bb_type_sd = import_fury_shader(os.path.join('billboard',
+                                                     'spherical.glsl'))
+        v_pos_mc = \
+            """
+            vec3 vertexPositionMC = sphericalVertexPos(center, MCVCMatrix,
+                                        normalizedVertexMCVSOutput, shape);
+            """
+        warnings.warn('Invalid option. The billboard will be generated '
+                      'with the default spherical option. ', UserWarning)
+
+    gl_position = \
+        '''
+        gl_Position = MCDCMatrix * vec4(vertexPositionMC, 1.);
+        '''
+
+    billboard_vert_impl = compose_shader(
+        [import_fury_shader('billboard_impl.vert') +
+         compose_shader(v_pos_mc) +
+         compose_shader(gl_position)])
+
     vs_dec_code = compose_shader(
-        [import_fury_shader('billboard_dec.vert') + compose_shader(vs_dec)]
-    )
+        [import_fury_shader('billboard_dec.vert') + compose_shader(vs_dec) +
+         compose_shader(bb_norm) +
+         compose_shader(bb_type_sd)])
     vs_impl_code = compose_shader(
-        [compose_shader(vs_impl) + import_fury_shader('billboard_impl.vert')]
-    )
+        [compose_shader(vs_impl) + billboard_vert_impl])
+
     gs_code = compose_shader(gs_prog)
     fs_dec_code = compose_shader(
         [import_fury_shader('billboard_dec.frag') + compose_shader(fs_dec)]
```

Comment:
Please add the content of `billboard_dec.frag`  to the python implementation and remove the file to comply with #753. Additionally, replace `+` with `,` when using the `compose_shader` function.